Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Transformer.purs

```diff
@@ -112,18 +112,20 @@ isEmailPresent :: LazyCheck -> Boolean
 isEmailPresent _ = not ( getValueToLocalStore USER_EMAIL == "__failed" || getValueToLocalStore USER_EMAIL == "(null)" )
 
 getApiIssueList :: Array IssueReportCustomerListItem -> Array IssueInfo
-getApiIssueList issueList = (map (\(IssueReportCustomerListItem issue) -> {
-   issueReportId : issue.issueReportId,
-   status : issue.status,
-   category : if (getLanguageLocale languageKey == "EN_US")
-                then
-                  joinWith " " (map (\catName ->
-                    let { before, after } = splitAt 1 catName
-                    in (toUpper before <> after)
-                  ) (split (Pattern " ") issue.category))
-                else issue.category,
-   createdAt : getExactTime (runFn2 differenceBetweenTwoUTC (getCurrentUTC "") (issue.createdAt)) issue.createdAt
-}) issueList)
+getApiIssueList issueList = map (\(IssueReportCustomerListItem issue) -> {
```

Comment:
This can be written as `map mapFunction issueList`

where
  mapFunction ::  --